Xamarin.Forms.Platform.iOS/Renderers/SliderRenderer.cs

```diff
@@ -44,11 +44,74 @@ protected override void OnElementChanged(ElementChangedEventArgs<Slider> e)
 				UpdateMaximum();
 				UpdateMinimum();
 				UpdateValue();
+				UpdateSliderColors();
 			}
 
 			base.OnElementChanged(e);
 		}
 
+		private void UpdateSliderColors()
+		{
+			UpdateMinimumTrackColor();
+			UpdateMaximumTrackColor();
+			if (!string.IsNullOrEmpty(Element.ThumbImage))
+			{
+				UpdateThumbImage();
+			}
+			else
+			{
+				UpdateThumbColor();
+			}
+		}
+
+		private void UpdateMinimumTrackColor()
+		{
+			if (Element != null && Element.MinimumTrackColor != Color.Default)
```

Comment:
this will no-op if the value changes back to default. have a look at what other renderers do (usually capturing the default color in a field, so it can be restored) e.g. https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs#L205-L220

Reply:
okay. I will make this change for both platforms